local_planner/include/local_planner/local_planner_node.h

```diff
@@ -275,6 +275,7 @@ class LocalPlannerNode {
 
   geometry_msgs::TwistStamped vel_msg_;
   bool armed_, offboard_, mission_, new_goal_;
+  bool auto_takeoff_;
```

Comment:
There's no way these uninitialized variables can be read before they are set, right?

Maybe we should set them all to `false` by default?